ci/src/main/scala/org/typelevel/sbt/TypelevelCiPlugin.scala

```diff
@@ -169,13 +169,13 @@ object TypelevelCiPlugin extends AutoPlugin {
             "validate-steward",
             "Validate Steward Config",
             WorkflowStep.Checkout ::
+              WorkflowStep.SetupJava(List(JavaSpec.temurin("17"))) :::
```

Comment:
... probably easiest to fix that by just setting this to 11.

Reply:
Updated